boards/renesas/rcar_h3ulcb/doc/rcar_h3ulcb_r7.rst

```diff
@@ -205,19 +205,19 @@ Flashing
 
 First of all, open your serial terminal.
 
-Applications for the ``rcar_h3ulcb_cr7`` board configuration can be built in the usual way (see :ref:`build_an_application` for more details).
+Applications for the ``rcar_h3ulcb/r8a77951/r7`` board configuration can be built in the usual way (see :ref:`build_an_application` for more details).
 
 .. zephyr-app-commands::
    :zephyr-app: samples/hello_world
-   :board: rcar_h3ulcb_cr7
+   :board: rcar_h3ulcb/r8a77951/r7
```

Comment:
`boards/renesas/rcar_h3ulcb/doc/rcar_h3ulcb_a57.rst` should have same changes

Reply:
Updated